docs/collections/documents/tutorials/bulk-update-restql.md

```diff
@@ -350,42 +407,71 @@ const jsc8 = require('jsc8');
+// Variables
+const email = "nemo@nautilus.com";
+const password = "xxxxxx";
 const collectionName = "superhero" + Math.floor(1000 + Math.random() * 9000).toString();
+const jamesKey = "james.kirk" + Math.floor(1000 + Math.random() * 9000).toString();
+const bruceKey = "bruce.wayne" + Math.floor(1000 + Math.random() * 9000).toString();
+const barryKey = "barry.allen" + Math.floor(1000 + Math.random() * 9000).toString();
 
 const inputDocs = [
-    { "_key": "james.kirk@mafabriccrometa.io", "firstname": "James", "lastname": "Kirk", "email": "james.kirk@mafabriccrometa.io", "zipcode": "12312"},
-    { "_key": "han.solo@macrfabricometa.io", "firstname": "Han", "lastname": "Solo", "email": "han.solo@macrfabricometa.io", "zipcode": "12311"},
-    { "_key": "bruce.wayne@mfabricacrometa.io", "firstname": "Bruce", "lastname": "Wayne", "email": "bruce.wayne@mfabricacrometa.io", "zipcode": "12345" }
+  { "_key": jamesKey, "firstname": "James", "lastname": "Kirk", "email": "james.kirk@macrometa.io", "zipcode": "12312" },
+  { "_key": bruceKey, "firstname": "Bruce", "lastname": "Wayne", "email": "bruce.wayne@macrometa.io", "zipcode": "12345" }
 ];
 
-const updateKeys = ["james.kirk@mafabriccrometa.io", "bruce.wayne@mfabricacrometa.io"];
+const updateKeys = [jamesKey, bruceKey];
 const updateKeyValue = {
-    "bruce.wayne@mfabricacrometa.io": { key: "bruce.wayne@mfabricacrometa.io", zipcode: "22222" },
-    "james.kirk@mafabriccrometa.io": { key: "james.kirk@mafabriccrometa.io", zipcode: "55555"}
+  bruceKey: { key: bruceKey, zipcode: "22222" },
+  jamesKey: { key: jamesKey, zipcode: "55555" }
 };
 
-//Queries
+// Queries
 const insertData = `FOR doc in @InputDocs \
-INSERT {"firstname":doc.firstname, "lastname":doc.lastname, "email":doc.email, "zipcode":doc.zipcode, "_key": doc._key} IN ${collectionName}`;
+INSERT { "firstname":doc.firstname, "lastname":doc.lastname, "email":doc.email, "zipcode":doc.zipcode, "_key": doc._key } IN ${collectionName}`;
 
 const getData = `FOR doc IN ${collectionName} RETURN doc`;
 
 const updateData = `FOR i IN ${collectionName} \
 FILTER i._key IN @updateKeys \
 UPDATE i with { zipcode: (i._key == @updateKeyValue[i._key].key) ? @updateKeyValue[i._key].zipcode : i.zipcode } IN ${collectionName}`;
 
-async function restqldemo() {
+const updatedInsertData = `INSERT 
+{ "firstname": "Barry", "lastname": "Allen", "email": "barry.allen@macrometa.io", "zipcode": "44444", "_key": @barryKey } IN ${collectionName}`;
+
+function errorResponseHandler (error) {
+  const message = {
+    "StatusCode ": error.statusCode,
+    "ErrorMessage ": error.message,
+    "ErrorNum ": error.errorNum
+  };
+  console.error(message);
+}
+
+const sleep = (milliseconds) => {
+  return new Promise(resolve => setTimeout(resolve, milliseconds))
+}
+
+async function restqldemo () {
+  try {
     /* Authenticate client instance with username and password */
     console.log("------- AUTHENTICATE CLIENT INSTANCE WITH USERNAME AND PASSWORD  ------");
```

Comment:
Why is this log and the next all caps when previous ones are in sentence case?

Reply:
I will change that caps ones to sentence case